### ISSUE TITLE
Multiple proxy services

### ISSUE THREAD

**Issue by arseniybanayev** [hamburml/docker-flow-letsencrypt#49]
Thank you for the wonderful service - I use it in production and it works smoothly.

Docker Flow Proxy supports running multiple independent proxy services (see `PROXY_INSTANCE_NAME `at https://proxy.dockerflow.com/config/). Would it be easy for you to support multiple `PROXY_ADDRESS` values?

**Comment by arseniybanayev** [comment]
Did it myself - https://github.com/hamburml/docker-flow-letsencrypt/pull/50

**Comment by arseniybanayev** [comment]
Ping @hamburml

**Comment by hamburml** [comment]
Thanks for the PR and the issue.

I don't use Docker Flow anymore and I don't have a setup to test this PR. The changes look right, but I can't be sure that certbot api is still the same. Did you test your changes?

**Comment by arseniybanayev** [comment]
Yes but only manually. I didn’t go through any testing suite. I have been using it in one of my projects with the changes and they work correctly.

Arseniy Banayev
arseniy.banayev@gmail.com
(407) 462-0998
________________________________

**Comment by hamburml** [comment]
I merged it into master.
You could try tag `latest` and if it works I'll tag it.
